ddtrace/appsec/_asm_request_context.py

```diff
@@ -122,25 +113,25 @@ def __init__(self):
 
         self._id = _DataHandler.main_id
         self.active = True
-        self.token = _ASM.set(env)
+        self.execution_context = core.ExecutionContext(__name__, **{"asm_env": env})
```

Comment:
Why expand the dict rather than just pass it in as a normal kwarg? 
```suggestion
        self.execution_context = core.ExecutionContext(__name__, "asm_env"= env)
```
Is it just a preference that doesn't matter?